addons/tutorial_04_split_tunneling/manifest.json

```diff
@@ -0,0 +1,94 @@
+{
+  "api_version": "0.1",
+  "id": "tutorial_04_split_tunneling",
+  "name": "Tutorial: Split tunneling",
+  "conditions": {
+    "enabled_features": ["splitTunnel"],
+    "javascript": "splitTunnelingOff.js"
+  },
+  "type": "tutorial",
+  "translatable": false,
+  "tutorial": {
+    "id": "04_split_tunneling",
+    "settings_rollback_needed": true,
+    "image": "qrc:/addons/tutorial_04_split_tunneling/image.svg",
+    "title": "How to exclude apps from VPN protection",
+    "subtitle": "Follow this walkthrough to learn how to exclude apps from VPN.",
+    "completion_message": "You’ve learned how to exclude apps from VPN protection. Would you like to learn more tips and tricks?",
+    "steps": [
+      {
+        "id": "s0",
+        "element": "controllerToggle",
+        "tooltip": "Toggle this switch to deactivate the VPN",
+        "conditions": {
+           "javascript": "vpnIsOn.js"
+        },
+        "next": {
+          "op": "signal",
+          "vpn_emitter": "controller",
+          "signal": "stateChanged"
+        }
+      }, {
+        "id": "s1",
+        "element": "navigationLayout/navButton-settings",
+        "tooltip": "TODO TODO",
```

Comment:
We shouldn't expose TODO strings.

Reply:
the addon manifest is marked as "translatable" false because of this. As soon as we have the final strings I mark the addon as translatable.

Reply:
Thanks, I forgot about that flag. The rest looks good.